app/views/computacenter/api/cap_usage/bulk_update.xml.builder

```diff
@@ -0,0 +1,17 @@
+xml.instruct!
+xml.CapUsageResponse('payloadId' => @batch.payload_id, 'dateTime' => Time.zone.now.iso8601) do
+  xml.HeaderResult('status' => @batch.status) do
+    xml.FailedRecords do
+      @batch.updates.select(&:failed?).each do |update|
```

Comment:
The start of this line would be cleaner and more explicit if pushed into `CapUsageUpdateBatch`:

```ruby
class CapUsageUpdateBatch
  ...
  def failed_updates
    updates.select(&:failed?)
  end
end
```

Reply:
done!